app/models/manageiq/providers/container_manager.rb

```diff
@@ -87,11 +87,17 @@ def port_show
       port.to_s
     end
 
+    # the purpose of this is to call:
+    #   - monitoring_manager.create_unless_exists
+    #   - infra_manager.build_unless_exists
```

Comment:
I think I'd word this slightly differently, what you have there is what this method _does_, the _purpose_ is to create/destroy child managers which are tied to an endpoint in a container_manager.